mm/kasan/kasan.c

```diff
@@ -101,7 +101,8 @@ static void kasan_report(FAR const void *addr, size_t size, bool is_write)
   if (++recursion == 1)
     {
       _alert("kasan detected a %s access error, address at %0#"PRIxPTR
```

Comment:
can we change to %p instead casting?

Reply:
The thing with `%p` is that its formatting is implementation specific, so I guess the original author chose `PRIxPTR` to enforce a specific formatting.

I am fine with whichever. Whatever you chose.